plugins/catalog-react/src/routes.ts

```diff
@@ -17,22 +17,18 @@
 import { Entity, ENTITY_DEFAULT_NAMESPACE } from '@backstage/catalog-model';
 import { createRouteRef } from '@backstage/core-plugin-api';
 
-const NoIcon = () => null;
-
 // TODO(Rugvip): Move these route refs back to the catalog plugin once we're all ported to using external routes
 export const rootRoute = createRouteRef({
-  icon: NoIcon,
-  path: '',
-  title: 'Catalog',
+  id: 'catalog',
 });
+
 export const catalogRouteRef = rootRoute;
 
 export const entityRoute = createRouteRef({
-  icon: NoIcon,
-  path: ':namespace/:kind/:name/*',
-  title: 'Entity',
+  id: 'catalog/entity',
```

Comment:
What do you think about `catalog` + `catalog-entity` for these instead and use that as a general pattern? And then use `/` for `SubRouteRef`s

Could also use some other separator like `catalog_entity` or `catalog:entity`, maybe the latter?